src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -4,6 +4,105 @@
 -->
 
 <docs>
+```vue
+<template>
+	<div class="styleguide-nc-content">
+		<NcAppNavigation>
+			<template>
+				<div class="navigation__header">
+					<NcTextField :value.sync="searchValue" label="Search …" />
+					<NcActions>
+						<NcActionButton close-after-click @click="showModal = true">
+							<template #icon>
+								<IconCog />
+							</template>
+							App settings (close after click)
+						</NcActionButton>
+						<NcActionButton @click="showModal = true">
+							<template #icon>
+								<IconCog />
+							</template>
+							App settings (handle only click)
+						</NcActionButton>
+					</NcActions>
+				</div>
+			</template>
+			<template #list>
+				<NcAppNavigationItem v-for="item in items" :key="item" :name="item">
+					<template #icon>
+						<IconCheck :size="20" />
+					</template>
+				</NcAppNavigationItem>
+			</template>
+			<template #footer>
+				<div class="navigation__footer">
+					<NcButton wide @click="showModal = true">
+						<template #icon>
+							<IconCog />
+						</template>
+						App settings
+					</NcButton>
+					<NcModal v-if="showModal" name="Modal for focus-trap check" @close="showModal = false">
+						<div class="modal-content">
+							<h4>Focus-trap should be locked inside the modal</h4>
+							<NcTextField :value.sync="modalValue" label="Focus me" />
```

Comment:
```suggestion
							<NcTextField v-model="modalValue" label="Focus me" />
```
And others. I'll handle it